### ISSUE TITLE
ACT with custom table don't work with SQLite Integration

### ISSUE THREAD

**Issue by tiptronic** [pods-framework/pods#3336]
I’m using with the ‘SQLite Integration’ plugin (http://dogwood.skr.jp/wordpress/sqlite-integration/), which is great, since my sites get much more portable when using SQLite (vs. MySQL).

Unfortunately  ‘Advanced Content Types’ don't work in PODS when using SQLite Integration (otherwise it works great). ACTs are never written to the database (and no table is created), which is a pity, because PODS and SQLite Integration are a VERY POWERFUL combo. The problem seems to be somewhere in the table-creation code...

I know SQLite is not _officially_ supported, but since it works just fine with PODS, it would be a huge benefit, if the problem using ACTs could be fixed.

**Comment by tiptronic** [comment]
If you need some help testing, please let me know... everything else but ACTs seems to work just fine.
I can get it working, when creating a custom table manually, but it is a bit ugly since I have to mess with SQLites auto-incrementing the UniqueKey

**Comment by tiptronic** [comment]
What does that mean? Is 'Enhancement Icebox' the road to death?

**Comment by sc0ttkclark** [comment]
Current goal is for us to close issues and focus on what we prioritize. There are many issues marked as feature / enhancement that are not yet prioritized. For those issues (including this one), we have tagged it as "Needs Votes". We then can look over features and enhancements over time to pick out the ones with the most demand and prioritize that way. It's something I recently saw used on some other large GitHub projects, it seemed to help them prioritize and focus on what's important.

This is now the issue query we use to prioritize upcoming features and enhancements for future releases that are not yet roadmapped:

https://github.com/pods-framework/pods/issues?q=label%3A%22Needs+Votes%22+is%3Aclosed+sort%3Areactions-%2B1-desc

https://github.com/pods-framework/pods/issues?q=label%3A%22Needs+Votes%22+is%3Aclosed+sort%3Acomments-desc
